apps/daffio/src/app/content/pwa/pages/pwa/pwa.component.html

```diff
@@ -0,0 +1,159 @@
+          <p daffFeatureSubtitle>
+            A community-driven list of stats and news focused on Progressive Web Apps.
+          </p>
+        </daff-feature>
+      </a>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      Who's building PWAs?
+    </h3>
+    <p daffCalloutSubtitle>
+      Brands are increasingly utilizing Progressive Web Apps to bridge the gap between native apps and websites to create better user experiences.
+    </p>
+    <div class="card-grid">
+      <a href="https://eng.uber.com/m-uber/" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/uber-logo.svg" />
+          <h4 daffCardTitle>Uber</h4>
+          <p>Uber built their PWA to minimize the client footprint while maintaining the rich experience of the native app.</p>
+        </daff-card>
+      </a>
+      <a href="https://app.starbucks.com/" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/starbucks-logo.svg" />
+          <h4 daffCardTitle>Starbucks</h4>
+          <p>The Starbucks PWA makes their service universally accessible to both established and emerging markets.</p>
+        </daff-card>
+      </a>
+      <a href="https://lancome-usa.com" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/lancome-logo.svg" />
+          <h4 daffCardTitle>Lanc&ocirc;me</h4>
+          <p>Lanc&ocirc;me's PWA aimed to drive traffic and re-engagement, which ultimately saw a 17% increase in conversions.</p>
+        </daff-card>
+      </a>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered" class="comparison">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      The best solution for ecommerce
+    </h3>
+    <p daffCalloutSubtitle>
+      PWAs are a rapidly developing technology that is continuously seeing new progressive enhancements. It is on the fast-track to becoming the new standard for mobile-first solutions.
+    </p>
+    <div class="comparison__grid">
+      <daffio-feature-comparison></daffio-feature-comparison>
+      <daff-list type="multi-line" class="comparison__list">
+        <daff-list-item>
+          <h4>Lower cost of production and maintenance</h4>
+          <p>Built as webpages, PWAs cost of development is much lower than native apps. Independent of operating systems or marketplaces, updates are done automatically and PWAs eliminate the need for super-specialized developers.</p>
+        </daff-list-item>
+        <daff-list-item>
+          <h4>Business Independence</h4>
+          <p>PWAs are free of marketplace fees and control. There is no 30% commission fee and they are free to deploy updates and promotions quickly and on short notice.</p>
```

Comment:
I feel like "quickly and on short notice" could be shortened to either "quickly" or "on short notice", since they mean the same thing.

Reply:
Resolved 5adc1d6